Cargo.toml

```diff
@@ -9,6 +9,9 @@ edition = "2018"
 serde = { version = "1.0", features = ["derive"] }
 #math = { git = "https://github.com/roadrunner-craft/math" }
 math = { path = "../math" }
+log = { version = "0.4.8", features = ["std"] }
+colored = "1.9"
```

Comment:
we should use `https://docs.rs/termcolor/1.1.0/termcolor/enum.Color.html` to get the rgb colors instead of being limited to the basic colors